src/js/letters/components/InvalidAddress.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const InvalidAddress = () => (
+  <div className="step-content">
+    <div id="invalidAddress">
+      <div className="usa-alert usa-alert-error">
+        <div className="usa-alert-body">
+          <h2 className="usa-alert-heading">Address unavailable</h2>
+          <p className="usa-alert-text">
+            We’re encountering an error with your address information. This is not required for your letters, but if you’d like to see the address we have on file, or to update it, please visit <a href="/" target="_blank">this link</a>.
```

Comment:
This isn't a new change, I know, but I don't like having a dangling link that goes nowhere helpful.

Do we have a ticket to track this? I feel like I wrote one up, but I can't seem to find it...

Reply:
Hmmm. @rroueche ? Do you know if we're tracking empty links somewhere?

Reply:
We should probably make sure we have these all covered across the app...

Reply:
Opened new issue to track this: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5004